src/btf.cpp

```diff
@@ -720,13 +788,31 @@ std::map<std::string, std::vector<std::string>> BTF::get_params(
     if (std::ranges::all_of(funcs, all_resolved))
       break;
 
-    auto mod_params = get_params_from_btf(btf_obj, funcs, is_raw_tracepoint);
+    auto mod_params = get_param_btf_cb(btf_obj, funcs);
     params.insert(mod_params.begin(), mod_params.end());
   }
 
   return params;
 }
 
+FuncParamLists BTF::get_params(const std::set<std::string> &funcs) const
+{
+  return get_params_impl(
+      funcs, [this](const BTFObj &btf_obj, const std::set<std::string> &funcs) {
+        return get_params_from_btf(btf_obj, funcs);
+      });
```

Comment:
Doesn't this work?
```suggestion
  return get_params_impl(funcs, get_params_from_btf);
```

Reply:
No. It's a non-static member function. I think the other option is to use `std::bind`